Mage/Task/BuiltIn/Deployment/Strategy/GitRebaseTask.php

```diff
@@ -53,8 +53,10 @@ public function run()
         $result = $this->runCommandRemote($command);
 
         if ($result === false) {
-            $repository = $this->getConfig()->deployment('repository');
-            if ($repository) {
+            $scmConfig = $this->getConfig()->deployment('scm');
```

Comment:
I wasn't sure about that. I can't find any reference to 'repository' in the example configuration or anywhere else for that matter. The scm/source key is used for the task mentioned before, so I assumed that was the correct value.